tests/unit/test_federal_senate_dataset.py

```diff
@@ -15,88 +16,95 @@ def setUpClass(cls):
 
             self.assertIn(expected_file, retrieved_file)
 
-    def test_fetch_not_found_files_from_S3(self):
-        self.path = gettempdir()
-        self.subject = Dataset(self.path, 2007, 2008)
+    @patch('serenata_toolbox.federal_senate.dataset.urlretrieve')
+    def test_fetch_raises_HTTPError(self, mocked_url_retrieve):
+        mocked_url_retrieve.side_effect = urllib.error.HTTPError(None, None, None, None, None)
+        path = gettempdir()
+        subject = Dataset(path, 2007, 2008)
+
+        with self.assertRaises(urllib.error.HTTPError) as context:
+            subject.fetch()
+
+        self.assertTrue(isinstance(context.exception, urllib.error.HTTPError))
 
-        retrieved_files, not_found_files = self.subject.fetch()
+    @patch('serenata_toolbox.federal_senate.dataset.urlretrieve')
+    def test_fetch_raises_URLError(self, mocked_url_retrieve):
+        mocked_url_retrieve.side_effect = urllib.error.URLError('tests reason')
+        path = gettempdir()
+        subject = Dataset(path, 2007, 2008)
 
-        for not_found_file, expected_file in zip(
-                not_found_files, self.expected_files):
+        with self.assertRaises(urllib.error.URLError) as context:
+            subject.fetch()
 
-            self.assertIn('federal-senate-2007.csv', not_found_file)
+        self.assertTrue(isinstance(context.exception, urllib.error.URLError))
 
     def test_dataset_translation(self):
-        self.subject = Dataset(os.path.join('tests', 'fixtures', 'csv'),
-                                            2008,
-                                            2009)
+        subject = Dataset(os.path.join('tests', 'fixtures', 'csv'),
+                          2008,
+                          2009)
 
         expected_files = ['federal-senate-2008.csv']
 
-        translated_files, not_found_files = self.subject.translate()
+        translated_files, _ = subject.translate()
 
         for translated_file, expected_file in zip(
                 translated_files, expected_files):
 
             self.assertIn(expected_file, translated_file)
 
     def test_if_translation_happened_as_expected(self):
-        self.subject = Dataset(os.path.join('tests', 'fixtures', 'csv'),
-                                            2008,
-                                            2009)
+        subject = Dataset(os.path.join('tests', 'fixtures', 'csv'),
+                          2008,
+                          2009)
 
-        file_path = os.path.join(self.subject.path, 'federal-senate-2008.csv')
+        file_path = os.path.join(subject.path, 'federal-senate-2008.csv')
         federal_senate_2008 = pd.read_csv(file_path,
                                           sep=';',
                                           encoding='ISO-8859-1',
                                           skiprows=1)
         self.assertIsNotNone(federal_senate_2008['ANO'],
                              'expects \'ANO\' as column in this dataset')
```

Comment:
Following [PEP8 on string quotes](https://www.python.org/dev/peps/pep-0008/#string-quotes):
>When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string

So let's remove this nasty backslash 😬